examples/all-clusters-app/nrfconnect/main/include/Utils.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
Just align all headers to the same pattern. There are too many changes to comment every file :smile: